python3/vimspector/variables.py

```diff
@@ -371,25 +470,40 @@ def ExpandVariable( self ):
       },
     } )
 
-  def _DrawVariables( self, view,  variables, indent ):
+  def _DrawVariables( self, view, variables, indent, is_short = False ):
```

Comment:
I will revert this function to its original state

Reply:
nvm, i didn't notice it was used inside `_DrawWatchResult`. not going to touch this.